x/supplier/client/cli/helpers_test.go

```diff
@@ -2,17 +2,39 @@
 package cli_test
 
 import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
 	"strconv"
 	"testing"
 
+	"cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	types3 "github.com/cosmos/cosmos-sdk/codec/types"
```

Comment:
NIT: Introduce semantically named types

Reply:
Nice catch, this was probably my IDE doing something dumb when I refactored some things.